CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v1.7.2 / 2019-08-05
+
+* [CHANGE] Revert "add kube_*_annotations metrics for all objects" #859
+* [CHANGE] Remove kube_namespace_annotations metric #859
```

Comment:
Why the Italics here?

Reply:
or rather the lack of italics in the previous line :).

Reply:
Aha I think its because of the above commit msg, the `_*_`